util/deadlockhistory/deadlock_history.go

```diff
@@ -72,18 +71,11 @@ func NewDeadlockHistory(capacity uint) *DeadlockHistory {
 	}
 }
 
-// globalDeadlockHistory is the global instance of DeadlockHistory, which is used to maintain recent several recent
+// GlobalDeadlockHistory is the global instance of DeadlockHistory, which is used to maintain recent several recent
 // deadlock events globally.
-var globalDeadlockHistory = NewDeadlockHistory(10)
+var GlobalDeadlockHistory = NewDeadlockHistory(10)
```

Comment:
When will the capacity be set when the server is just started, and the config file are set to a different value?

Reply:
I'll put the init code in `setGlobalVars` in `tidb-server/main.go`.